cmd/podman/remoteclientconfig/configfile_test.go

```diff
@@ -92,14 +92,15 @@ func TestReadRemoteConfig(t *testing.T) {
 		{"nouser", args{reader: strings.NewReader(noUser)}, makeNoUserResult(), false},
 	}
 	for _, tt := range tests {
+		test := tt
```

Comment:
why keep 'tt' at all, can't you drop this line and replace tt in line 94 with 'test'?

Reply:
We shouldn't use loop variables in go which may leak outside the loop (e.g., by appending it to a slice outside of the loop) or when the code can run in parallel. The latter is the case for all t.Run(..)s as unit tests could run in parallel.

Reply:
The "problem" with loop variables is that they're actually a pointer. `test := tt` allocates a new variable and copies the value of `tt`. This way, we're safe.

Reply:
thx, didn't see the leak outside of the loop happening in this code, but may well have missed it.